src/processes/scoring/calc-score-on-follow-user.js

```diff
@@ -51,5 +47,5 @@ const calcScoreOnFollowUser = async (
 };
 
 module.exports = {
-  calcScoreOnFollowUser,
+  calcScoreOnFollowUser
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a minor issue with the export statement. The trailing comma is not necessary when exporting a single function.
```diff
-  calcScoreOnFollowUser
+  calcScoreOnFollowUser
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->